src/lib/pkg.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable import/order -- some rules are in conflict, but it's fine */
+import type { PackageJson } from 'type-fest';
+
+export const name =
```

Comment:
This is nice! Would this approach work for getting the package version too?

Reply:
this gets _this_ package info, it's bandled by webpack. the tricky bit is getting webpack not to try to optimise away the `require` that loads package of the source this action is checking

Reply:
i bet there's a npm module that would handle this though...

Reply:
Ah yes, fair enough. I've got a separate point on the checklist to improve the way we get the version number so let's leave it for now and pick it up as it's own task later.